blockpage_signatures.json

```diff
@@ -451,131 +451,186 @@
-{"pattern":"Blocked-By: NextDNS'","fingerprint":"satellite_nextdns"}
-{"pattern":"<small>The website/domain has been blocked as per order of Ministry of Electronics and Information Technology under IT Act, 2000</small>","fingerprint":"satellite_india1"}
-{"pattern":"<title>Internet Protect</title>","fingerprint":"satellite_indonesia20"}
-{"pattern":"<title>Blocked Content</title>","fingerprint":"satellite_indonesia21"}
-{"pattern":"<title>StopPage</title>","fingerprint":"satellite_belguim2"}
-{"pattern":"<title>Доступ закрыт</title>","fingerprint":"satellite_russia13"}
-{"pattern":"<title>Technitium DNS Server</title>","fingerprint":"satellite_nepal_technitium"}
-{"pattern":"<title>Website durch Pi-hole blockiert!</title>","fingerprint":"satellite_pihole"}
-{"pattern":"<title>Block-Content</title>","fingerprint":"satellite_nepal4"}
-{"pattern":"This domain is blocked<br>Ce domaine est bloqué","fingerprint":"satellite_canada"}
-{"pattern":"This Page has Been Blocked<BR>","fingerprint":"satellite_indonesia22"}
-{"pattern":"<title>Maaf, Situs ini tidak bisa di akses</title>","fingerprint":"satellite_indonesia23"}
-{"pattern":"<title>OneDNS block system</title>","fingerprint":"satellite_onedns"}
-{"pattern":"<FONT COLOR=\"#AA4444\">Съгласно Закон за Хазарта и разпореждания на Софийски Районен Съд,</FONT>","fingerprint":"satellite_bulgaria1"}
-{"pattern":"<TITLE>BLOCKED</TITLE>","fingerprint":"satellite_general"}
-{"pattern":"<title>Витрина</title>","fingerprint":"satellite_bulgaria2"}
-{"pattern":"<title>Malicious Website blocked</title>","fingerprint":"satellite_general2"}
-{"pattern":"<h2>Website ini memiliki larangan akses</h2>","fingerprint":"satellite_indonesia24"}
-{"pattern":"pursuant to an Order of the High Court of Singapore","fingerprint":"satellite_singapore"}
-{"pattern":"<title>Block !</title>","fingerprint":"satellite_general3"}
-{"pattern":"<title>Resource is blocked</title>","fingerprint":"satellite_general4"}
-{"pattern":"<body id=\"blockpage\">","fingerprint":"satellite_pihole2"}
-{"pattern":"var pname='internet-sehat';","fingerprint":"satellite_indonesia25"}
-{"pattern":"URL=http://sehat.hypernet.co.id\"","fingerprint":"satellite_indonesia26"}
-{"pattern":"<!-- Pi-hole: A black hole for Internet advertisements","fingerprint":"satellite_pihole3"}
-{"pattern":"Mohon maaf untuk saat ini akses internet anda sedang terblokir<br>","fingerprint":"satellite_indonesia27"}
-{"pattern":"<form id=\"block_public\" name=\"block_public\" action=\"/block.php\" method=\"GET\">","fingerprint":"satellite_italy10"}
\ No newline at end of file
+
+{"fingerprint":"a_prod_adguard_satellite","pattern":"blocking-pages - Blocking pages for AdGuard projects"}
+{"fingerprint":"a_prod_blue_shield_satellite","pattern":"<title>Blocked by BLUE SHIELD v3</title>"}
+{"fingerprint":"a_prod_guardiandns_satellite","pattern":"Guardian DNS"}
+{"fingerprint":"a_prod_id_skynet_satellite","pattern":"<title>SkyNet \\| Internet Sehat</title>"}
```

Comment:
Skynet seems to be an ISP?

Reply:
Yes you are right. https://sky.net.id/ 
Interesting that Indonesia and Russia both have a Skynet ISP.